packages/core/src/dropdown/Dropdown.css

```diff
@@ -49,52 +65,103 @@
 
 .saltDropdown-primary {
   --dropdown-background: var(--salt-editable-primary-background);
+  --dropdown-background-active: var(--salt-editable-primary-background-active);
+  --dropdown-background-hover: var(--salt-editable-primary-background-hover);
+  --dropdown-background-disabled: var(--salt-editable-primary-background-disabled);
+  --dropdown-background-readonly: var(--salt-editable-primary-background-readonly);
 }
 
 .saltDropdown-secondary {
   --dropdown-background: var(--salt-editable-secondary-background);
+  --dropdown-background-active: var(--salt-editable-secondary-background-active);
+  --dropdown-background-hover: var(--salt-editable-secondary-background-active);
+  --dropdown-background-disabled: var(--salt-editable-secondary-background-disabled);
+  --dropdown-background-readonly: var(--salt-editable-secondary-background-readonly);
 }
 
-.saltDropdown:disabled {
-  --dropdown-background: var(--salt-editable-primary-background-disabled);
+.saltDropdown:disabled,
+.saltDropdown:disabled:active,
+.saltDropdown:disabled:hover,
+.saltDropdown:disabled:focus {
+  --dropdown-background: var(--dropdown-background-disabled);
   --dropdown-borderColor: var(--salt-editable-borderColor-disabled);
   --dropdown-color: var(--salt-content-primary-foreground-disabled);
   --dropdown-cursor: var(--salt-selectable-cursor-disabled);
+  --dropdown-borderWidth: var(--salt-size-border);
+  outline: none;
```

Comment:
```suggestion
```

Reply:
If this is needed, then dropdown disabled managed is wrong

Reply:
removed this.disabled has no outline now.